City-of-Helsinki/drupal-helfi-platform-config
# UHF-4444: News feed paragraph

# [UHF-4444](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4444)

## How to install
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-4444`
* Run `drush en helfi_news_feed`
* Add `$config['helfi_news_feed.settings']['source_environment'] = 'dev';` to your `local.settings.php` file.

## How to test
* Add `News feed` paragraph in finnish Landing page and make sure you only see finnish news
* Translate the Landing page and make sure you only see english news
